cli/pcluster/cfnconfig.py

```diff
@@ -804,7 +804,14 @@ def __init_fsx_parameters(self):
             return
 
         # Check that the base_os is supported
-        self.__validate_os("FSx", self.__get_os(), ["centos7"])
+        self.__validate_os("FSx", self.__get_os(), ["centos7", "alinux"])
+
+        # Validate scheduler is not awsbatch
+        if self.parameters.get("Scheduler") == "awsbatch":
+            self.__fail(
+                "FSx isn't supported with awsbatch as the scheduler, please submit a feature request"
```

Comment:
* minor: _awsbatch as scheduler_?
* you need to add a space after the _request_ word because the message is printed in a single line.

Reply:
fixed